src/modules/app/i18n.js

```diff
@@ -0,0 +1,37 @@
+import i18n from 'i18next'
+import LanguageDetector from 'i18next-browser-languagedetector'
+import localesResources from '../../locales'
+import { reduce, forEach } from 'lodash/collection'
+
+const FALLBACK_LANGUAGE = 'en'
+
+/**
+ * i18n is an instance of i18next which is responsible for language detection and change, translations and rtl vs ltr layout
```

Comment:
:warning: Warning, Warning! Comment in the wild!

Is the comment below suggesting to add back the I18n class? I think that would be good.

Reply:
We have to fix the whole i18n thing, I would do this in a separate task